ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml

```diff
@@ -6869,48 +6835,35 @@ periodics:
       - --lease-server-password-file=/etc/boskos/password
       - --lease-server-username=ci
       - --secret-dir=/usr/local/pull-secret
-      - --secret-dir=/usr/local/e2e-azure-upgrade-cluster-profile
-      - --target=e2e-azure-upgrade
+      - --secret-dir=/usr/local/e2e-$(CLUSTER_TYPE)-upgrade-cluster-profile
+      - --target=e2e-$(CLUSTER_TYPE)-upgrade
```

Comment:
These should remain as `azure`. The `$(CLUSTER_TYPE)` is `azure4`, which is different than the target and secret names. `pj-rehease` is failing trying to handle this job, so I think changing this back should fix the rehearsal.